include/ncdispatch.h

```diff
@@ -9,7 +9,9 @@
 #ifndef _DISPATCH_H
 #define _DISPATCH_H
 
+#if HAVE_CONFIG_H
```

Comment:
We always have config.h, right?

Reply:
This is a common practice when using autotools.